query_optimizer/PhysicalGenerator.cpp

```diff
@@ -108,6 +109,7 @@ P::PhysicalPtr PhysicalGenerator::generateInitialPlan(
 P::PhysicalPtr PhysicalGenerator::optimizePlan() {
   std::vector<std::unique_ptr<Rule<P::Physical>>> rules;
   rules.emplace_back(new PruneColumns());
+  rules.emplace_back(new PushDownLowCostDisjunctivePredicate());
```

Comment:
Oh I made a mistake that actually the `InjectFilterJoin` optimization will not generate unnecessary `Selection`s, but `PushDownLowCostDisjunctivePredicate` may generate two `Selection`s in a chain. Currently this is a not an issue since the `Selection`s for tables with cardinality around 100 are very light-weight. We may add a `FuseSelection` optimization later.